CMakeLists.txt

```diff
@@ -198,10 +198,6 @@ add_library(s2
             src/s2/util/math/exactfloat/exactfloat.cc
             src/s2/util/math/mathutil.cc
             src/s2/util/units/length-units.cc)
-add_library(s2testing STATIC
```

Comment:
gtest requires c++14 or higher.

Reply:
Disabling for now, since currently experimenting with ubuntu 24 upgrade of the base image.